tests/foreman/ui/test_sync.py

```diff
@@ -40,19 +37,22 @@ def test_sync_custom_repos(self, repository_name):
         """
         # Creates new product
         product = entities.Product(organization=self.organization).create()
-        # Creates new repository
-        entities.Repository(
-            name=repository_name,
-            url=FAKE_1_YUM_REPO,
-            product=product,
-        ).create()
         with Session(self.browser) as session:
-            session.nav.go_to_select_org(self.organization.name)
-            session.nav.go_to_sync_status()
-            sync = self.sync.sync_custom_repos(
-                product.name, [repository_name])
-            # syn.sync_custom_repos returns boolean values and not objects
-            self.assertTrue(sync)
+            for repository_name in generate_strings_list():
+                with self.subTest(repository_name):
+                    # Creates new repository
```

Comment:
As we are putting API code into `with Session` section, maybe it will be better to clarify that moment
```
# Creates new repository through API
```